python/setup.py

```diff
@@ -28,8 +28,8 @@
     packages=['googledatastore'],
     package_dir={'googledatastore': 'googledatastore'},
     install_requires=[
-        'httplib2==0.8',
-        'oauth2client==2.0.1',
+        'httplib2>=0.9.1,<0.10',
+        'oauth2client>=2.0.1,<4.0.0',
```

Comment:
Do we need the upper bounds? I'm not sure how to go about testing that we're compatible with all of these major versions.